.github/workflows/snapshot-tests.yaml

```diff
@@ -12,6 +13,8 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
+        with:
+          ref: develop
```

Comment:
Scheduled workflows run on the latest commit on the default branch, which is the `main` branch in our case.